packages/neos-ui-redux-store/src/UI/AddNodeModal/index.js

```diff
@@ -59,25 +60,32 @@ export const reducer = handleActions({
     [system.INIT]: () => $set(
         'ui.addNodeModal',
         new Map({
-            referenceNode: '',
+            reference: null,
```

Comment:
hm, what is "reference"? I find it's hard to understand, because it is unclear what is in here and why we need it.